galaxy_ng/app/migrations/0025_add_content_guard_to_distributions.py

```diff
@@ -0,0 +1,32 @@
+from django.db import migrations
+
+
+def add_content_guard(apps, schema_editor):
+    """
+    Enforce that all distributions have a content guard applied
+    """
+    AnsibleDistribution = apps.get_model('ansible', 'AnsibleDistribution')
+    ContentRedirectContentGuard = apps.get_model('galaxy', 'ContentRedirectContentGuard')
+
+    content_guard, _ = ContentRedirectContentGuard.objects.get_or_create(pulp_type='ansible.ansible')
```

Comment:
I'd give this a name. It looks like if you leave out the name it defaults to "", which is no good.